### ISSUE TITLE
Adjusting initial LR when changing batch size. 

### ISSUE THREAD

**Issue by eoinoconn** [ultralytics/yolov5#5472]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

I am in a position where I can up to double my BS for training the YoloV5 net, however its unclear to me how I should adjust the initial LR. A quick search unearthed this question which highlights some good rules of thumb for increasing LR with BS. However I thought YoloV5 may be different due to the scaling of the loss by BS. Any advice?

https://stackoverflow.com/questions/53033556/how-should-the-learning-rate-change-as-the-batch-size-change

**Comment by glenn-jocher** [comment]
@eoinoconn YOLOv5 automatically adjusts training settings, no action is required on your part to use different batch sizes other than to simply pass whatever you want to --batch

**Comment by eoinoconn** [comment]
@glenn-jocher what adjustments does yolo make? I can see it adjusts weight decay accordingly but it lr altered by batch-size? Is the adjustment linear?

**Comment by glenn-jocher** [comment]
@eoinoconn all adjustments are made automatically, there is no action for you to take if you adjust batch size.

**Comment by eoinoconn** [comment]
@glenn-jocher I understand adjustments are automatic but understanding what the adjustments are would be useful.

**Comment by glenn-jocher** [comment]
@eoinoconn yes I understand but I don't have availability to explain everything to everyone. All batch size updates are in train.py and loss.py. If you find any bugs or want to suggest features please let us know.
